site/app/controllers/DockerInterfaceController.php

```diff
@@ -153,11 +152,45 @@ public function addImage() {
         }
     }
 
-    /**
-     * @return JsonResponse
-     */
+    #[Route("/admin/remove_image", methods: ["POST"])]
+    public function removeImage(): JsonResponse {
+        $pattern = '/^[a-z0-9]+[a-z0-9._(__)-]*[a-z0-9]+\/[a-z0-9]+[a-z0-9._(__)-]*[a-z0-9]+:[a-zA-Z0-9][a-zA-Z0-9._-]{0,127}$/';
+        $user = $this->core->getUser();
+        $user_id = $user->getId();
```

Comment:
```suggestion
```

Unused variable

Reply:
Good catch. Surprised that didn't get caught by the static analysis.